### PR TITLE
Update text on about/faq, home, and registration pages.

### PR DESCRIPTION
**Pull request by maurizi**
Connects to #1727

### CODE REVIEW

File: src/nyc_trees/apps/home/templates/home/about_faq.html [hunk 1]
```diff
@@ -12,93 +12,103 @@
...
 <p>
-    At the training event, we&rsquo;ll have
-    staff teaching you how to apply what you&rsquo;ve learned, answer any
-    questions you have, supply necessary mapping tools and a Tree Species
-    ID guide, and get you mapping right away with other newly trained
-    volunteers!
-    Start your online training <a href="{% url 'training_summary_pure' %}">here</a>!</p>
+    Mapping events are intended for volun<b>treers</b> who have already been to a training event.
+    No instruction period is included, but staff will be present to help you out if you get stuck.
+    When you arrive, you will be paired with another volunteer, check out equipment using your volun<b>treer</b> badge,
```
Comment (RickMohr) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: Should that be "another volun**treer**"?

File: src/nyc_trees/apps/home/templates/home/about_faq.html [hunk 2]
```diff
@@ -12,93 +12,103 @@
...
-    Just
-    a love of trees and a willingness to learn! At the training event,
-    you will receive a TreesCount! Gear Pack with a Tree Species ID guide
-    and a measuring tape, along with a volunteer badge and lanyard, for
-    you to keep and reuse throughout the duration of the census. NYC
-    Parks and our partners will provide you with a measuring wheel and a
-    tablet for mapping trees and collecting data. These will be collected
-    at the end of the event.</p>
+    Just a love of trees and a willingness to learn! You will need your TreesCount! username
+    and password in order to be checked into the event. At the training event, you will receive
```
Comment (RickMohr) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: Should that be "you will receive a TreesCount! Gear Pack"?

File: src/nyc_trees/apps/home/templates/home/about_faq.html [hunk 3]
```diff
@@ -12,93 +12,103 @@
...
-        instruction period is included; volun<b>treers</b>
-            will start mapping right away!
+    We are hosting events in neighborhoods throughout all five boroughs of New York City.
+    Many local organizations are contributing to the mapping effort, and you will be able to
+    participate in events hosted by TreesCount! partner groups and by NYC Parks.
+    After you’ve completed your online training, find training and mapping events near you <a href="{% url "events_list_page" %}">here</a>.
 </p>
 
-<h4>Do I need to bring anything to a training event?</h4>
+<h4>What should I bring to a mapping event?</h4>
```
Comment (maurizi) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: This header should stay as "Do I need to bring anything to a training event?"

File: src/nyc_trees/apps/home/templates/home/about_faq.html (hunk 2 above)
Comment (maurizi) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: Fixed

File: src/nyc_trees/apps/home/templates/home/about_faq.html (hunk 1 above)
Comment (maurizi) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: That is not what was indicated in the text provided to us, so I am going to leave this as-is.

**Comment by RickMohr** [comment]
+1

**Pull request merged**